diffpy/pdfmorph/pdfmorphapp.py

```diff
@@ -62,9 +61,58 @@ def custom_error(self, msg):
     parser.add_option(
         '-s',
         '--save',
-        metavar="FILE",
-        dest="savefile",
-        help="Save manipulated PDF from FILE1 to FILE. Use \'-\' for stdout.",
+        metavar="SLOC",
+        dest="saveloc",
+        help="""Save the manipulated PDF to a file SLOC. Use \'-\' for stdout.
+When --multiple is enabled, each manipulated PDF is saved to a directory SLOC.
+Names for each manipulated PDF can be specified using --snf.""",
+    )
+    parser.add_option(
+        '--snf',
+        '--save-names-file',
+        metavar="SNFILE",
+        dest="snfile",
+        help="""Used only when -s and --multiple are enabled.
```

Comment:
be clear if this is OR or AND, e.g., "Used only if both -s and -multiple are enabled."

Again, read it with a beginning grad in chemistry in mind....will it be clear to them what to input here and when and how?  Examples are sometimes better than text filled with jargon.

Reply:
Changed the names and descriptions. Also added an example for ``--save-names-file` in tutorial.